examples/scenarioSepMomentumManagement.py

```diff
@@ -353,20 +353,62 @@ def run(momentumManagement, cmEstimation, showPlots):
     lenYHub = 1.8  # [m]
     lenZHub = 2.86  # [m]
     area2 = np.pi*(0.5 * 7.262)*(0.5 * 7.262)  # [m^2]
-    facetAreas = [lenYHub * lenZHub, lenXHub * lenZHub, lenYHub * lenZHub, lenXHub * lenZHub, lenXHub * lenYHub, lenXHub * lenYHub, area2, area2, area2, area2]
-
-    # Define the facet normals in B frame components
-    facetNormal1 = np.array([1.0, 0.0, 0.0])
-    facetNormal2 = np.array([0.0, 1.0, 0.0])
-    facetNormal3 = np.array([-1.0, 0.0, 0.0])
-    facetNormal4 = np.array([0.0, -1.0, 0.0])
-    facetNormal5 = np.array([0.0, 0.0, 1.0])
-    facetNormal6 = np.array([0.0, 0.0, -1.0])
-    facetNormal7 = np.array([0.0, 1.0, 0.0])
-    facetNormal8 = np.array([0.0, -1.0, 0.0])
-    facetNormal9 = np.array([0.0, 1.0, 0.0])
-    facetNormal10 = np.array([0.0, -1.0, 0.0])
-    normals_B = [facetNormal1, facetNormal2, facetNormal3, facetNormal4, facetNormal5, facetNormal6, facetNormal7, facetNormal8, facetNormal9, facetNormal10]
+    facetAreaList = [lenYHub * lenZHub,
+                     lenXHub * lenZHub,
+                     lenYHub * lenZHub,
+                     lenXHub * lenZHub,
+                     lenXHub * lenYHub,
+                     lenXHub * lenYHub,
+                     area2,
+                     area2,
+                     area2,
+                     area2]
+
+    # Define the initial facet attitudes relative to B frame
+    prv_F01B = (macros.D2R * -90.0) * np.array([0.0, 0.0, 1.0])
+    prv_F02B = (macros.D2R * 0.0) * np.array([0.0, 0.0, 1.0])
+    prv_F03B = (macros.D2R * 90.0) * np.array([0.0, 0.0, 1.0])
+    prv_F04B = (macros.D2R * 180.0) * np.array([0.0, 0.0, 1.0])
+    prv_F05B = (macros.D2R * 90.0) * np.array([1.0, 0.0, 0.0])
+    prv_F06B = (macros.D2R * -90.0) * np.array([1.0, 0.0, 0.0])
+    prv_F07B = (macros.D2R * 0.0) * np.array([0.0, 0.0, 1.0])
+    prv_F08B = (macros.D2R * 180.0) * np.array([0.0, 0.0, 1.0])
+    prv_F09B = (macros.D2R * 0.0) * np.array([0.0, 0.0, 1.0])
+    prv_F010B = (macros.D2R * 180.0) * np.array([0.0, 0.0, 1.0])
+    facetDcm_F0BList = [rbk.PRV2C(prv_F01B),
+                        rbk.PRV2C(prv_F02B),
+                        rbk.PRV2C(prv_F03B),
+                        rbk.PRV2C(prv_F04B),
+                        rbk.PRV2C(prv_F05B),
+                        rbk.PRV2C(prv_F06B),
+                        rbk.PRV2C(prv_F07B),
+                        rbk.PRV2C(prv_F08B),
+                        rbk.PRV2C(prv_F09B),
+                        rbk.PRV2C(prv_F010B)]
+
+    # Define the facet normal vectors in B frame components
```

Comment:
Is this in F or B frame components? I think the comment should be updated

Reply:
Yes, thank you! Just completed this.